projects/planner/src/app/diver/diver.component.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { DecimalPipe } from '@angular/common';
+import { DiverComponent } from './diver.component';
+import { UnitConversion } from '../shared/UnitConversion';
+import { ValidatorGroups } from '../shared/ValidatorGroups';
+import { InputControls } from '../shared/inputcontrols';
+import { DiverOptions } from '../shared/models';
+
+describe('DiverComponent', () => {
+    let component: DiverComponent;
+    let fixture: ComponentFixture<DiverComponent>;
+    let unitConversion: UnitConversion;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            declarations: [DiverComponent],
+            imports: [ReactiveFormsModule],
+            providers: [
+                UnitConversion,
+                ValidatorGroups,
+                InputControls,
+                DecimalPipe
+            ]
+        }).compileComponents();
+
+        fixture = TestBed.createComponent(DiverComponent);
+        component = fixture.componentInstance;
+        unitConversion = TestBed.inject(UnitConversion);
+        component.diver = new DiverOptions();
+        fixture.detectChanges();
+    });
+
+    describe('inputChanged', () => {
+        it('should change rmv and stressRmv values and emit', () => {
+            expect(component.diver.rmv).toBe(20);
+            expect(component.diver.stressRmv).toBe(30);
+
+            const newRmv = 15;
+            const newStressRmv = 25;
+
+            component.diverForm.setValue({ rmv: newRmv, stressRmv: newStressRmv });
```

Comment:
Dont use the form directly, it should be used for bindings only, use PageObject instead.